src/core/ast.ml

```diff
@@ -496,7 +496,7 @@ let unescape s =
 				| 'r' -> Buffer.add_char b '\r'
 				| 't' -> Buffer.add_char b '\t'
 				| '"' | '\'' | '\\' -> Buffer.add_char b c
-				| '0'..'3' ->
+				| '0'..'1' ->
```

Comment:
Maybe it should still be parsed as `'0'...'1'` but error on `> 1`. To avoid silently changing behavior if someone is actually using this feature.